Codecademy/gamut
# feat(RatingsBar): create RatingsBar gamut-labs component

### Overview


Adds a RatingsBar component to gamut-labs.

To be used for the ratings distribution summary on the revamped course landing page.

### PR Checklist

- [x] Related to designs: https://www.figma.com/file/mAzeq3F7H9Ior4wCIL0oZl/Experiments---Ratings-and-Reviews?node-id=640%3A74455&t=pYyeJGq6C3ypdZIj-0
- [x] Related to JIRA ticket: [REACH-2240](https://codecademy.atlassian.net/browse/REACH-2240)

#### Testing Instructions

1. Visit `/?path=/docs/brand-labs-atoms-ratingsbar--ratings-bar`.
2. Confirm there is a story for the `RatingsBar` and that it shows an example of one (which looks similar to the progress bar with a yellow bar on a white background).
![Screen Shot 2022-11-16 at 12 33 29 PM](https://user-images.githubusercontent.com/29311886/202289241-cdf59b9c-4363-4dbd-8cc5-9cdc7c2d6abc.png)
3. Turn on VoiceOver. Tab into the example of the ratings bar. Confirm the screenreader announces the ratings bar as "75% of ratings".

#### PR Links and Envs
NOTE: This component is to be used in portal-app for the revamped  course landing page.

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/585)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-594)   |
